validation/src/main/scala/hmda/validation/rules/lar/quality/Q029.scala

```diff
@@ -0,0 +1,19 @@
+package hmda.validation.rules.lar.quality
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+import hmda.validation.dsl.PredicateGeo._
+
+object Q029 extends EditCheck[LoanApplicationRegister] {
+  override def apply(lar: LoanApplicationRegister): Result = {
+    val geo = lar.geography
+    when(geo is shouldHaveMsa) {
```

Comment:
Hm. This doesn't quite read well, and it puts the logic of the rule outside of the rule. (If I understand correctly, the only reason it should have an MSA is because of Q029.) Is there a way to name the predicate that describes what it actually checks? (And, perhaps more challenging: ... while not being ridiculously long?)